metron-platform/metron-management/src/main/java/org/apache/metron/management/EnrichmentConfigFunctions.java

```diff
@@ -0,0 +1,351 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.metron.management;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.jakewharton.fliptables.FlipTable;
+import org.apache.log4j.Logger;
+import org.apache.metron.common.configuration.FieldTransformer;
+import org.apache.metron.common.configuration.enrichment.EnrichmentConfig;
+import org.apache.metron.common.configuration.enrichment.SensorEnrichmentConfig;
+import org.apache.metron.common.dsl.Context;
+import org.apache.metron.common.dsl.ParseException;
+import org.apache.metron.common.dsl.Stellar;
+import org.apache.metron.common.dsl.StellarFunction;
+import org.apache.metron.common.utils.ConversionUtils;
+import org.apache.metron.common.utils.JSONUtils;
+
+import java.util.*;
+
+import static org.apache.metron.common.configuration.ConfigurationType.ENRICHMENT;
+
+public class EnrichmentConfigFunctions {
+
+  private static final Logger LOG = Logger.getLogger(ConfigurationFunctions.class);
+  public enum Type {
+    ENRICHMENT, THREAT_INTEL, THREATINTEL;
+  }
+  public static Map<String, Object> getStellarHandler(EnrichmentConfig enrichmentConfig) {
+    Map<String, Object> fieldMap = enrichmentConfig.getFieldMap();
+    Map<String, Object> stellarHandler = (Map<String, Object>) fieldMap.get("stellar");
+    if(stellarHandler == null ) {
+      stellarHandler = new HashMap<String, Object>();
+      fieldMap.put("stellar", stellarHandler);
+    }
+
+    if(stellarHandler.get("config") == null){
```

Comment:
Can we use `stellarHandler.putIfAbsent("config", new LinkedHashMap<String, Object>());`?  It's a Java 8 thing, and I haven't seen anyone suggest moving back to Java 7, so might as well get some use out of those functions.